src/js/controllers/auth/securityQuestionController.js

```diff
@@ -216,8 +217,9 @@
 
             $scope.initNavigator.on('postpush', () => {
                 $timeout(() => {
-                    const securityAnswer = document.getElementById('security_answer_input');
-                    if (securityAnswer) {
+                    const securityAnswer = document.getElementById('security-answer-input');
```

Comment:
This code can't find  the element, because the new `security-answer-input` ID is missing.

Reply:
Doh. Accidentally "fixed" the problem this way.